### PR TITLE
Create E45F01FFFE38DA63

### PR DESCRIPTION
**Pull request by umbertowski**
NEW GATEWAY

### CODE REVIEW

File: E45F01FFFE38DA63 [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "servers": [
+      {
+        "server_address": "eu1.cloud.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 44,
+    "ref_longitude": 10,
```
Comment (gonzalocasas) [inline E45F01FFFE38DA63:13]: Again, please use proper lat/long, this is too coarse.
Also, is the EUI really coming from the mac address of your network interface or did you make it up?

**Review by gonzalocasas — CHANGES_REQUESTED** [review]

File: E45F01FFFE38DA63 (hunk 1 above)
Comment (umbertowski) [reply E45F01FFFE38DA63]: i've update the lat and the long
eui
The EUI it coming from here

**Comment by umbertowski** [comment]
i've update it now

Il giorno lun 9 ago 2021 alle ore 09:35 Gonzalo Casas <
***@***.***> ha scritto:

**Comment by gonzalocasas** [comment]
Perfect! Thx a lot!

**Pull request merged**